tabular/src/autogluon/tabular/models/widedeep_nn/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,33 @@
+from autogluon.core import Categorical, Real, Int
+from autogluon.core.constants import BINARY, MULTICLASS, REGRESSION
+
+
+def get_default_searchspace(problem_type, num_classes=None):
+    if problem_type == BINARY:
+        return get_searchspace_binary().copy()
+    elif problem_type == MULTICLASS:
+        return get_searchspace_multiclass(num_classes=num_classes)
+    elif problem_type == REGRESSION:
+        return get_searchspace_regression().copy()
+    else:
+        return get_searchspace_binary().copy()
+
+
+def get_searchspace_binary():
+    spaces = {
+        # See docs: https://docs.fast.ai/tabular.models.html
+        'bs': Categorical(256, 64, 128, 512, 1024, 2048, 4096),
+        'lr': Real(5e-5, 1e-1, default=1e-2, log=True),
+        'epochs': Int(lower=5, upper=30, default=30),
```

Comment:
default should match actual default - currently 10.

Reply:
fixed